dev-client/src/model/soilIdMatch/soilIdMatches.ts

```diff
@@ -89,3 +93,15 @@ export const dataEntryForMatches = (
     matches: matches,
   };
 };
+
+export const flushErrorEntries = (
+  entries:
+    | Record<string, SoilIdLocationEntry>
+    | Record<string, SoilIdDataEntry>,
+) => {
+  for (const id of Object.keys(entries)) {
+    if (isErrorStatus(entries[id].status)) {
+      delete entries[id];
```

Comment:
Yes, because this method is invoked in the `deleteSiteMatches` action which operates on a draft of new state. It will never modify existing state. (Redux generally is structured to disallow unintended modification of state.)